src/commands/synthetics/README.md

```diff
@@ -307,22 +235,235 @@ yarn datadog-ci synthetics run-tests -f ./component-1/**/*.synthetics.json -f ./
 <!-- xxz tab xxx -->
 <!-- xxz tabs xxx -->
 
-### Failure modes flags
+### List of Configurations
+
+<!--
+  The descriptions of these is kept for consistency in this doc - https://datadoghq.atlassian.net/wiki/spaces/SYN/pages/3378446383/Configuration+parameters+and+their+usages#Proposal-for-aligning-Descriptions-and-labels
+  If want to update them please update the doc and the relevant integrations as well.
+-->
+
+#### `apiKey`
+
+The API key used to query the Datadog API.
+
+Configuration options:
+
+* Global Config: `"apiKey": "<API_KEY>"`
+* ENV variable: `DATADOG_API_KEY="<API_KEY>"`
+* CLI param: `--apiKey "<API_KEY>"`
+
+#### `appKey`
+
+The application key used to query the Datadog API.
+
+Configuration options:
+
+* Global Config: `"appKey": "<APPLICATION_KEY>"`
+* ENV variable: `DATADOG_APP_KEY="<APPLICATION_KEY>"`
+* CLI param: `--appKey "<APPLICATION_KEY>"`
+
+#### `batchTimeout`
+
+The duration (integer in milliseconds) after which `datadog-ci` stops waiting for test results. The default is 30 minutes. At the CI level, test results completed after this duration are considered failed.
+
+Configuration options:
+
+* Global Config: `"batchTimeout": 180000`
+* ENV variable: `DATADOG_SYNTHETICS_BATCH_TIMEOUT=180000`
+* CLI param: `--batchTimeout 180000`
+
+#### `configPath`
+
+The global JSON configuration is used when launching tests. See the [example configuration](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration/?tab=npm#global-configuration-file-options ) for more details.
+
+Configuration options:
+
+* Global Config: N/A
+* ENV variable: `DATADOG_SYNTHETICS_CONFIG_PATH=global-config.json`
+* CLI param: `--config global-config.json`
+
+#### `datadogSite`
+
+The Datadog instance to which request is sent. The default is `datadoghq.com`.<!-- partial Your Datadog site is {{< region-param key="dd_site" code="true" >}}. partial -->
+
+Configuration options:
+
+* Global Config: `"datadogSite": "datadoghq.com"`
+* ENV variable: `DATADOG_SITE=datadoghq.com`
+* CLI param: `--datadogSite datadoghq.com`
+
+#### `defaultTestOverrides`
+
+Overrides for Synthetic tests applied to all tests.
+
+Configuration options:
+
+* Global Config: See `testOverrides` part of [Test files](#test-files) for an example.
```

Comment:
This shows that we could improve the structure of the README: I think we should have a section named "Test overrides", which would list the possible test overrides

And it would say that those overrides can either be applied to all tests with `defaultTestOverrides`, or to some specific tests with their `testOverrides`.

The "Test files" section would then be smaller.